### PR TITLE
adding a readme

### PR DESCRIPTION
**Pull request by josh-robbs**
Request input review

Added a readme because reasons.

### CODE REVIEW

**Comment by WillKirkhope** [comment]
This is trash

**Comment by RJKirkhope** [comment]
Thank you SO much Timothy.

**Comment by WillKirkhope** [comment]
I hate both of you

File: readme.md [hunk 1]
```diff
@@ -0,0 +1 @@
+goddamnit RJ
```
Comment (RJKirkhope) [inline readme.md:1]: Thank you SO much Timothy.

Comment (WillKirkhope) [inline readme.md:1]: I think that this would be better "Thou hast been Fuched"

Comment (RJKirkhope) [reply readme.md:1]: Thanks for the feedback Timothy!

**Pull request merged**